plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/dialog/FileOpenSaveDialog.java

```diff
@@ -1024,6 +1051,14 @@ private Image rasterImage( String path, int width, int height ) {
     return image;
   }
 
+  private void setBtnUpEnable( IStructuredSelection structuredSelection ) {
+    if ( !structuredSelection.isEmpty() && structuredSelection.getFirstElement() instanceof Directory ) {
+      flatBtnUp.setEnabled( true );
```

Comment:
No issue with logic, just suggestion on coding style. If author don't agree, ignore.

no need for if/else statement

`flatBtnUp.setEnabled( <expression> )`

Reply:
Applied your suggestions, Thank you!